app/Http/Controllers/CompetitionsController.php

```diff
@@ -115,4 +116,18 @@ public function removeUser(Competition $competition, User $user)
 
         return redirect()->back()->with('status', 'User was removed from competition ' . $competition->id);
     }
+
+    /**
+     * Detach a user from a competition.
```

Comment:
What is this function for? Is it just showing all messages? should this be moved into the `MessageController`?